pkg/events/event_store.go

```diff
@@ -21,31 +21,31 @@ package events
 import (
 	"sync"
 
+	"go.uber.org/zap"
+
+	"github.com/apache/yunikorn-core/pkg/log"
 	"github.com/apache/yunikorn-core/pkg/metrics"
 	"github.com/apache/yunikorn-scheduler-interface/lib/go/si"
 )
 
-const defaultEventStoreSize = 1000
-
 // The EventStore operates under the following assumptions:
 //   - there is a cap for the number of events stored
 //   - the CollectEvents() function clears the currently stored events in the EventStore
 //
 // Assuming the rate of events generated by the scheduler component in a given time period
 // is high, calling CollectEvents() periodically should be fine.
 type EventStore struct {
-	events []*si.EventRecord
-	idx    int // points where to store the next event
+	events   []*si.EventRecord
+	idx      int // points where to store the next event
```

Comment:
this means we cannot use the fulll size as that is an unit64.Should we not have the same types?